dev/CommandBarFlyout/CommandBarFlyout_themeresources.xaml

```diff
@@ -624,6 +624,49 @@
         </Setter>
     </Style>
 
+    <contract7NotPresent:Style x:Key="CommandBarFlyoutCommandBarOverflowPresenterStyleRS4" TargetType="CommandBarOverflowPresenter">
```

Comment:
You can say `BasedOn="{StaticResource CommandBarFlyoutCommandBarOverflowPresenterStyle}"` - then the only thing you need to set is the Template property.  Will reduce duplication somewhat.